tests/CMakeLists.txt

```diff
@@ -235,16 +227,15 @@ if (USE_CUDA OR USE_HIP)
   else () # i.e. USE_HIP
 
     foreach (libsmm_acc_test ${LIBSMM_ACC_TESTS_SRCS})
-      set_source_files_properties(${libsmm_acc_test}
-                                  PROPERTIES HIP_SOURCE_PROPERTY_FORMAT 1)
 
       get_filename_component(libsmm_acc_test_name ${libsmm_acc_test} NAME_WE)
 
-      hip_add_executable(${libsmm_acc_test_name} ${libsmm_acc_test})
-      target_link_options(${libsmm_acc_test_name} PRIVATE ${HIP_ARCH_FLAGS})
-
-      target_link_libraries(${libsmm_acc_test_name} dbcsr)
+      add_executable(${libsmm_acc_test_name} ${libsmm_acc_test})
+      target_link_libraries(${libsmm_acc_test_name} dbcsr hip::host)
       target_compile_definitions(${libsmm_acc_test_name} PRIVATE __HIP)
+      if (OpenMP_FOUND)
+        target_link_libraries(${libsmm_acc_test_name} OpenMP::OpenMP_CXX)
```

Comment:
I haven't checked, but if the `libsmm_acc_test_name` sources (but it is required by `libsmm_acc`) do not explicitly use OpenMP then `OpenMP::OpenMP_CXX` should be exported by the the library and passed down via `target_link_libraries(... dbcsr)` rather than explicitly.

Reply:
I'll check